torch/csrc/jit/import.cpp

```diff
@@ -260,8 +260,10 @@ TypePtr ModuleDecoder::buildType(const onnx::TypeProto& type_proto) {
     return NoneType::get();
   } else if (kind == "GeneratorType") {
     return GeneratorType::get();
-  }else if (kind == "StringType") {
+  } else if (kind == "StringType") {
     return StringType::get();
+  } else if (kind.find("TypeVar:") == 0) {
+    return VarType::create(kind.substr(strlen("TypeVar:")));
```

Comment:
We shouldn't need export/import for `TypeVar`s right? They're compile time only

Reply:
That is correct today. But it seemed like less work to implement it than to insert the error message and we maybe need it in the future.